solana-programs/anchor/audius-data/tests/follows.ts

```diff
@@ -222,8 +225,64 @@ describe("follows", function () {
       expect(decodedData.followeeHandle.seed).to.deep.equal(
         constants2.handleBytesArray
       );
-      expect(accountPubKeys[0]).to.equal(adminStorageKeypair.publicKey.toString());
-      expect(accountPubKeys[3]).to.equal(newUser1Key.publicKey.toString());
+      expect(accountPubKeys[0]).to.equal(
+        adminStorageKeypair.publicKey.toString()
+      );
+      expect(accountPubKeys[5]).to.equal(newUser1Key.publicKey.toString());
+    });
+
+    it("delegate follows user", async function () {
+      const delegate = await testCreateUserDelegate({
+        adminKeypair,
+        adminStorageKeypair: adminStorageKeypair,
+        program,
+        provider,
+        ...getURSMParams(),
+      });
+
+      // Submit a tx where user 1 follows user 2
+      const followArgs = {
+        accounts: {
+          audiusAdmin: adminStorageKeypair.publicKey,
+          authority: delegate.userAuthorityDelegateKeypair.publicKey,
+          followerUserStorage: delegate.userAccountPDA,
+          followeeUserStorage: userStorageAccount2,
```

Comment:
seems like we're reusing the follower/followees created in the outer scope but there's no afterEach that resets the follow?

Reply:
ah is it just that we're creating a new set of users in each beforeEach